Rocket.Chat/Resources/de.lproj/Localizable.strings

```diff
@@ -154,11 +154,11 @@
 "subscriptions.list.no_message" = "Keine Nachricht";
 "subscriptions.list.sent_an_attachment" = "versende einen Anhang";
 "subscriptions.list.date.yesterday" = "Gestern";
-"subscriptions.list.actions.read" = "Read"; // TODO
-"subscriptions.list.actions.unread" = "Unread"; // TODO
-"subscriptions.list.actions.favorite" = "Favorite"; // TODO
-"subscriptions.list.actions.unfavorite" = "Unfavorite"; // TODO
-"subscriptions.list.actions.hide" = "Hide"; // TODO
+"subscriptions.list.actions.read" = "Lesen";
+"subscriptions.list.actions.unread" = "Ungelesen";
+"subscriptions.list.actions.favorite" = "Favorisieren";
+"subscriptions.list.actions.unfavorite" = "Defavorisieren";
```

Comment:
We may check if the action box allows text that long @rafaelks

Reply:
Thought ‘bout the same, but was to lazy to look up how WhatsApp and Co. handle it 😅

Reply:
There it is like "Ungele-sen", not pretty but anyhow...